src/domain/timeline/calendar/components/FullCalendar.tsx

```
@@ -216,15 +246,62 @@ export interface FullCalendarProps {
   sx?: BoxProps['sx'];
 }
 
+const addEventToBucket = (
+  obj: { [key: string]: FullCalendarProps['events'] } | {},
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)